solidity-checks/src/main/java/org/sonarsource/solidity/checks/FileRuleVerifier.java

```diff
@@ -74,5 +75,9 @@ public void addIssueOnFile(String reportMessage, String externalRuleKey) {
     protected void clearList() {
       issues.clear();
     }
+
+    @Override
+    public void addIssue(ParserRuleContext ctx, String reportMessage, String externalRuleKey) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. ([squid:S1186](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS1186))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=63&issues=AWSsvO-kl6vpliCet4fW&open=AWSsvO-kl6vpliCet4fW)

[ISSUE_KEY]: AWSsvO-kl6vpliCet4fW